hasura/README.md

```diff
@@ -13,9 +13,9 @@ and all of our backend services.
 
 1. `cp hasura/.env.example hasura/.env.local`
 1. In `hasura/.env.local` set the value of `HASURA_GRAPHQL_ADMIN_SECRET` to the Hasura admin secret value generated at the beginning of this setup process. Set the value of `EVENT_SECRET` to another secure random value.
-1. If you are using an M1 Mac, note that Hasura's official Docker image does not yet support this platform. You will need to use a different image built by a third party. Swap `image: hasura/graphql-engine:v{version number}.cli-migrations-v2` for `image: fedormelexin/graphql-engine-arm64:v{version number}.cli-migrations-v2` in `docker-compose.yaml`, replacing the version number as appropriate.
```

Comment:
Update the example envfile with the new Hasura version that works on M1 Macs - `hasura/graphql-engine:v2.2.0.cli-migrations-v3` IIRC.